components/gitpod-db/src/typeorm/migration/1669104291275-PersonalAccessTokenIndexCreatedAt.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { columnExists } from "./helper/helper";
+
+const table = "d_b_personal_access_token";
+const col = "createdAt";
+const indexName = "idx_createdAt";
+
+export class PersonalAccessTokenIndexCreationTime1669104291275 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        if (!(await columnExists(queryRunner, table, col))) {
```

Comment:
Is the negation correct here? I think you want to check that the column *does* exist.

Reply:
Good spot. I actually want to check that the index does not exist so this is all wrong..

Reply:
fixed, thanks!